README.rst

```diff
@@ -0,0 +1,218 @@
+Mail BB Gun
+===========
+
+A baby version of MailGun
+
+Requirements
+------------
+
+* Python 3.6
+* Docker
+* SMTP Server account
```

Comment:
This is an interesting question that sent me down a rabbit hole of Wikipedia articles on mail protocols and gave me a laundry list of RFCs to read.

I chose to use an SMTP account because I assume most people have one, and it was the path of least resistance to get a working solution quickly.

After being prompted by your question to look deeper into this, I found some other alternatives.  The one I found to be most interesting is to look up the MX records for the recipient's domain and talking to that SMTP server directly.